tests/lib/rules/object-curly-spacing.js

```diff
@@ -58,6 +58,9 @@ ruleTester.run("object-curly-spacing", rule, {
 
         // always - empty object
         { code: "var foo = {};", options: ["always"] },
+        { code: "var foo = {};", options: ["always", { spaceInEmptyObject: "never" }] },
+        { code: "var foo = { };", options: ["always", { spaceInEmptyObject: "always" }] },
```

Comment:
Could you add some tests for line breaks? (I assume those should always pass and should be handled by the `object-curly-newline` rule.)

```js
var foo = {
};
// Should pass(?) for all combinations of first option + spaceInEmptyObject option
```